### ISSUE TITLE
Improvements needed for network isolated

### ISSUE THREAD

**Issue by vbauzysvmware** [terraform-providers/terraform-provider-vcd#373]
* resource doesn't provide ability to provide description
* when Dns ip is wrong, error isn't precise:
```
Error: error: error creating the network: error instantiating a new OrgVDCNetwork: API Error: 400: [ afe4b21d-bee8-4e94-ac10-a228ca64cf32 ] validation error on field 'ipSettingList[0].nameServer1': String value has invalid format
```
* when one of pool address  is wrong then error isn't precise too:
```
Error: error: error creating the network: error instantiating a new OrgVDCNetwork: API Error: 400: [ 6b72cf9a-e95a-464f-bc76-0adbeff8e783 ] validation error on field 'ipSettingList[0].ipAddressRanges[0].startAddress': String value has invalid format
```
* dhcp_pool field with details isn't refreshed when dhcp pool settings enabled/disabled or ip range is removed in UI. This can be reproduced: create isolated netw, goto actions "configure services" :
![image](https://user-images.githubusercontent.com/41415459/67848932-4b1fcd00-fb0e-11e9-8686-1491d78998e5.png)

**Comment by dataclouder** [comment]
Please provide sample HCL scripts that reproduces the issues.

**Comment by vbauzysvmware** [comment]
`resource "vcd_network_isolated" "TestAccVcdNetworkIsoMixed" {
  name       = "TestAccVcdNetworkIsoMixed"
  org        = "my1"
  vdc        = "vbVdc"
  gateway    = "192.168.2.1111"
  dns1       = "192.168.2.1"
  static_ip_pool {
    start_address = "192.168.2.2222"
    end_address   = "192.168.2.50"
  }
  dhcp_pool {
    start_address = "192.168.2.51"
    end_address   = "192.168.2.100"
  }
}`

**Comment by dataclouder** [comment]
Fixed in #402
